src/Stratis.SmartContracts.Executor.Reflection/Executor.cs

```diff
@@ -0,0 +1,102 @@
+﻿using Microsoft.Extensions.Logging;
+using Stratis.SmartContracts.Core;
+using Stratis.SmartContracts.Core.State;
+
+namespace Stratis.SmartContracts.Executor.Reflection
+{
+    public class Executor : ISmartContractExecutor
```

Comment:
Hmmm not too sure about the name? hehehe it istoo agnostic @rowandh ?

Also, please write a short description on what the purpose of the class is 👍